packages/formstr-app/src/containers/FormFiller/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { V1FormSpec } from "@formstr/sdk/dist/interfaces";
+import StyleWrapperChild from "../CreateForm/components/QuestionsList/style";
+import StyleWrapper from "../CreateForm/index.style";
+import FormTitle from "../CreateForm/components/FormTitle";
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { getFormTemplate } from "@formstr/sdk";
+import { Form, Typography } from "antd";
+import { QuestionNode } from "./QuestionNode/QuestionNode";
+
+const { Text } = Typography;
+
+export const FormFiller = () => {
+  const { formId } = useParams();
+  const [formTemplate, setFormTemplate] = useState<V1FormSpec | null>(null);
+  const [form] = Form.useForm();
+
+  const handleInput = (
+    questionId: string,
+    answer: string,
+    message?: string
+  ) => {
+    form.setFieldValue(questionId, [answer, message]);
+  };
+
+  useEffect(() => {
+    async function getForm() {
+      if (!formTemplate) {
+        if (!formId) {
+          throw Error("Form Id not provided");
+        }
+        let form = await getFormTemplate(formId);
```

Comment:
set loading true until we get a response and show loader to user

Reply:
will add in a subsequent PR